Frontend/classico/README.md

```diff
@@ -0,0 +1,16 @@
+# classico
```

Comment:
Modify the default README of this git repo which is at parent directory level. README needs to be elaborate enough to explain the purpose, utility of your project and must provide all instructions to clone, run your project.